sample.ts

```diff
@@ -91,6 +91,24 @@ const sampleStyle: Style = {
         graphicFill: {
           kind: 'Icon'
         }
+      }, {
+        kind: 'Raster',
+        opacity: 1,
+        colorMap: {
+          colorMapEntries: [
+            {
+              color: '#00FFFF',
+              quantity: 100
+            }
+          ],
+          extended: false,
+          type: "ramp"
+        },
+        channelSelection: {
+          grayChannel: {
+            sourceChannelName: '1'
+          }
+        }
```

Comment:
Might be better to create a seperate sample file for raster styles as mixing doesn't make sense, or does it?

Reply:
According to the SLD Reference mixing is fine. Also checked it on geoserver and had no validation errors.

Reply:
I don't know any datasource that stores raster and vector data at the same time.
I think we also need an own `StyleType` for raster data `raster`, as mixing really makes no sense to me.
BTW: Do we use this property anywhere? :)

Reply:
> as mixing really makes no sense to me.

I agree with you on that. I can create another example, that's not a problem. Anyway, we should also support mixing the types as it is not our concern if that makes sense for a user or not.

> BTW: Do we use this property anywhere? :)

We use it at least in the ol-parser. Will add the according property.